package.json

```diff
@@ -19,5 +19,9 @@
   "bugs": {
     "url": "https://github.com/supersteve2019/Tanki/issues"
   },
-  "homepage": "https://github.com/supersteve2019/Tanki#readme"
+  "homepage": "https://github.com/supersteve2019/Tanki#readme",
+  "devDependencies": {
+    "mocha": "^6.2.0",
+    "must": "^0.13.4"
```

Comment:
@supersteve2019 The `package.json` file describes metadata about this project.

One of the bits of metadata is what other _modules_ we are using: we're using Node so we're able to benefit from the thousands of other modules that folk have created.

Whenever you see that the `package.json` file has been updated, you'll almost always want to install/update any new/updated modules. In this case I've added some dependencies — something that we _depend_ on to provide some useful stuff — to run automated tests.

Run the following command in your _shell_: you may also have heard me describe it as the _terminal_.

```shell
npm i
```

The above command will install any new/updated dependencies from the Internet.